src/material/datepicker/calendar-body.scss

```diff
@@ -175,6 +174,12 @@ $mat-calendar-range-end-body-cell-size:
   // Choosing a value clearly larger than the height ensures we get the correct capsule shape.
   border-radius: $mat-calendar-body-cell-radius;
 
+  // Apply this style with additional specificity to override the focus indicator's relative
+  // positioning.
+  .mat-calendar-body-cell & {
+    position: absolute;
```

Comment:
Again, including this style as I'm operating under the assumption that we can't rely upon stylesheet ordering. This style now has specificity 0,2,0, while the global styles have 0,1,0.

Reply:
Spoke offline, we can rely upon stylesheet ordering, so removed this change.